src/common/exif.cc

```diff
@@ -1378,6 +1378,7 @@ int dt_exif_read_blob(uint8_t *buf, const char *path, const int imgid, const int
       memcpy(buf + 6, &(blob[0]), length);
       return length + 6;
     }
+    fprintf(stderr, "[exif] Warning: too much data to store (%d bytes), Exif data will be discarded\n", length);
```

Comment:
In that case, maybe we could let the data blob be set regardless of size and wait for failure at write? The issue here is that it's quite hard to track absence of exif data since it's rejected in a read function...